__snapshots__/expansion panel.md

```diff
@@ -7,14 +7,17 @@
   <button
     aria-controls="content"
     class="expansion-panel-button"
+    id="expansion-panel"
   >
     <mwc-ripple>
     </mwc-ripple>
     <span class="leading-icon">
       <slot name="icon">
       </slot>
     </span>
-    click me
+    <span class="heading-text">
```

Comment:
it's no longer heading text right? it's the button's label.
it's implementation details and not an issue but maybe deserves a better name for maintainers legibility

Reply:
It is the button label - but in the expansion-panel is is the headline of the expansion. 
Its only a class name for our use.

Reply:
I know, missed it in the previous PR. NM